src/main/java/ca/uwaterloo/drinkmasterapi/feature/mqtt/model/PourItem.java

```diff
@@ -0,0 +1,37 @@
+package ca.uwaterloo.drinkmasterapi.feature.mqtt.model;
+
+// not used for mqtt
+public class PourItem {
+
+    private int id;
+    private float amount;
```

Comment:
use double instead

Reply:
that is what chad ask for, and this model is not-used, is an sample here, could be discussed lateron